src/isEqual.ts

```diff
@@ -0,0 +1,47 @@
+import { Narrow } from './_narrow';
+import { purry } from './purry';
+
+/**
+ * Returns a function that determines if a value is _shallowly_ equal to the specified value using Object.is equality,
```

Comment:
No need to mention that it returns a function; that is what the dataLast designation means.
```suggestion
 * Determines if a value is _shallowly_ equal to the specified value using Object.is equality,
```